internal/spdxlicense/generate/generate_license_list.go

```diff
@@ -139,9 +187,26 @@ func processSPDXLicense(result LicenseList) map[string]string {
 		}
 	}
 
+	for k, v := range licenseIDs {
+		cleanID := strings.ToLower(v)
+		if remap, exists := overwrite[cleanID]; exists {
+			licenseIDs[k] = remap
+		}
+	}
+
 	return licenseIDs
 }
 
+func findEquivalentLicense(deprecated License, allLicenses []License) *License {
```

Comment:
this seems important enough to have an explicit unit test for this (especially when adding the extra condition in)

Reply:
done